NotificationBanner/Classes/BaseNotificationBanner.swift

```diff
@@ -58,13 +58,25 @@ open class BaseNotificationBanner: UIView {
             if let customBannerHeight = customBannerHeight {
                 return customBannerHeight
             } else {
-                return shouldAdjustForNotchFeaturedIphone() ? 88.0 : 64.0 + heightAdjustment
+                return basicHeight()
```

Comment:
I do not see the `shouldAdjustForNotchFeaturedIphone()` anymore, but was not able to see any bug in the application

Reply:
Yes, this `shouldAdjustForNotchFeaturedIphone` was not right, because worked only for phone in portrait mode, also this calculation was not working right bor bottom inset when the banner is shown from the bottom. So, right now instead of it `basicYOffset` is used and it is under the hood added in `basicHeight` function.